helm/prometheus-operator-app/values.yaml

```diff
@@ -39,29 +39,32 @@ prometheus-operator-app:
       ingressClassName: nginx
       pathType: ImplementationSpecific
   prometheusOperator:
-    enabled: true
```

Comment:
I remember if this flag is omit, there's a condition about NetworkPolicy will fail!